### PR TITLE
Major rename and cleaning

### PR DESCRIPTION
**Pull request by dryanmedia**
- NOTE: PULLED #10 and #24 into this PR and closed them. Too much overlapping.
- RENAMED files
- RENAMED functions
- IMPROVED comments
- LEFT functions in existing positions for moment (want better file/directory org)
- LEFT all options alone (have documented all settings we'll have to write database upgrade script for. *Going to make a ticket but doesn't look difficult to mirror Will's method of applying db updates.*

### CODE REVIEW

**Comment by dryanmedia** [comment]
TO-DO
- [ ] Move plugin initialization into link-roundups.php. (For clarity, lroundups is being used for functionality directly related to the link roundups post type and mailchimp. link-roundups is being used globally for the plugin).
- [ ] Better organize functions (separate backwards compatibility file)
- [ ] Check for plain text and make everything translatable strings with $domain vars

File: js/lroundups.js [hunk 1]
```diff
@@ -65,22 +65,22 @@ var AL = AL || {};
...
                 '.admin.mailchimp.com/campaigns/wizard/confirm?id=' + model.get('web_id') +
                 '">Click here to edit your campaign in MailChimp.</a>';
 
-            return AL.Modal.prototype.initialize.apply(this, arguments);
+            return LR.Modal.prototype.initialize.apply(this, arguments);
         }
     });
 
     var updateCreateCampaignButton = function(model) {
         var markup = '<p>A MailChimp roundup campaign exists for thist post.</p>' +
```
Comment (benlk) [inline js/lroundups.js]: Is there another PR that fixes the `thist` typo?

**Comment by aschweigert** [comment]
i don't believe so

File: saved-links-class.php [hunk 2]
```diff
@@ -528,42 +528,54 @@ public static function the_excerpt($content) {
...
-?>
-	  <p class='link-roundup'>
+		$style = $link_style; // $link_style is defined below in rounduplink_shortcode()
+		
+		ob_start(); 
+		
+		/* begin shortcode output
+		 *
+		 * NOTE: 
+		 * This default output is overwritten 98% of the time by $argo_html
```
Comment (benlk) [inline saved-links-class.php]: Should `$argo_html` be `$lroundups_html` here?

Comment (dryanmedia) [inline saved-links-class.php]: Yes should be $lroundups_html.

**Comment by benlk** [comment]
Can #29 be rolled into this as well? It's entirely about cleaning up the tabs and spaces of indenting.

**Comment by benlk** [comment]
Two bugs in this PR: 

- The "Send to Editor" button in this PR's version of the plugin doesn't do anything. It neither creates an error in the console nor moves the link to the editor.
- Running the update script does not cause the "Run the update" notification box to go away. It disappears on the update page, but continues to appear on other pages afterwards.

**Comment by dryanmedia** [comment]
Per chat with @rnagle:

- [ ] Better method for sponsored link class & test with MC
- [ ] Send to Editor bug
- [ ] Update bug 

Then I'm going to do:
- [ ] One last read through of each file

Then we can close this pull and focus on more discrete pulls :+1:

**Comment by dryanmedia** [comment]
The Mailchimp CSS Inliner is what we'll need to use to target css classes in the Mailchimp campaign. This will have to be done for each client / site using the plugin. 
http://kb.mailchimp.com/campaigns/ways-to-build/use-the-css-inliner

**Comment by dryanmedia** [comment]
Not sure I can pass the word "Sponsored" in easily to Mailchimp the same way I am for the post. Need to send a test email but need to config a live server.

https://github.com/INN/link-roundups/pull/41/files#diff-95e51e37919bdf29323da9b9e9b3303eR97

**Comment by benlk** [comment]
The most-recent commits move the saved links to the bottom of the main column of meta boxes.

Also: The "Send to Editor" button is broken again. :disappointed: 

screen shot 2015-07-29 at 10 02 19 am

**Comment by dryanmedia** [comment]
sigh... i had it working in 85aef32 just a couple commits back...

**Pull request merged**